modules/core/src/bitgo.ts

```diff
@@ -1282,687 +1626,626 @@ BitGo.prototype.listAccessTokens = function(params, callback) {
-* @param {boolean} cpfpAware flag indicating fee should take into account CPFP
-* @returns
-*/
-BitGo.prototype.estimateFee = function(params, callback) {
-  params = params || {};
-  common.validateParams(params, [], [], callback);
-
-  const queryParams: any = { version: 12 };
-  if (params.numBlocks) {
-    if (!_.isNumber(params.numBlocks)) {
-      throw new Error('invalid argument');
+  /**
+   * Get the blockchain object.
+   * @deprecated
+   */
+  blockchain(): any {
+    if (!this._blockchain) {
+      this._blockchain = new Blockchain(this);
     }
-    queryParams.numBlocks = params.numBlocks;
+    return this._blockchain;
   }
-  if (params.maxFee) {
-    if (!_.isNumber(params.maxFee)) {
-      throw new Error('invalid argument');
+
+  /**
+   * Get the user's keychains object.
+   * @deprecated
+   */
+  keychains(): any {
+    if (!this._keychains) {
+      this._keychains = new Keychains(this);
     }
-    queryParams.maxFee = params.maxFee;
+    return this._keychains;
   }
-  if (params.inputs) {
-    if (!Array.isArray(params.inputs)) {
-      throw new Error('invalid argument');
+
+  /**
+   * Get the user's wallets object.
+   * @deprecated
+   */
+  wallets() {
+    if (!this._wallets) {
+      this._wallets = new Wallets(this);
     }
-    queryParams.inputs = params.inputs;
+    return this._wallets;
   }
-  if (params.txSize) {
-    if (!_.isNumber(params.txSize)) {
-      throw new Error('invalid argument');
+
+  /**
+   * Get the travel rule object
+   * @deprecated
+   */
+  travelRule(): any {
+    if (!this._travelRule) {
+      this._travelRule = new TravelRule(this);
     }
-    queryParams.txSize = params.txSize;
+    return this._travelRule;
   }
-  if (params.cpfpAware) {
-    if (!_.isBoolean(params.cpfpAware)) {
-      throw new Error('invalid argument');
+
+  /**
+   * Get pending approvals that can be approved/ or rejected
+   * @deprecated
+   */
+  pendingApprovals(): any {
+    if (!this._pendingApprovals) {
+      this._pendingApprovals = new PendingApprovals(this);
     }
-    queryParams.cpfpAware = params.cpfpAware;
+    return this._pendingApprovals;
   }
 
-  return this.get(this.url('/tx/fee'))
-  .query(queryParams)
-  .result()
-  .nodeify(callback);
-};
+  /**
+   * A factory method to create a new Wallet object, initialized with the wallet params
+   * Can be used to reconstitute a wallet from cached data
+   * @param walletParams
+   * @deprecated
+   */
+  newWalletObject(walletParams): any {
+    return new Wallet(this, walletParams);
+  }
 
-//
-// instantGuarantee
-// Get BitGo's guarantee using an instant id
-//
-BitGo.prototype.instantGuarantee = function(params, callback) {
-  params = params || {};
-  common.validateParams(params, ['id'], [], callback);
+  /**
+   * Create a url for calling BitGo platform APIs
+   * @param path
+   * @param version
+   */
+  url(path: string, version = 1): string {
```

Comment:
```version: number = 1``` ?

Reply:
Yeah. Unfortunately this needs to stay 1, since a lot of v1 code calls this with the default 2nd arg.

Reply:
oh I'm saying there's no type on it though